source/workflows/vod_translate_workflow.yaml

```diff
@@ -0,0 +1,133 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
What's the difference between this workflow and the MieCompleteWorkflowTranslate workflow?